api/models/post.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const postSchema = new Schema({
+    postDate: {
+        type: Date,
+        default: Date.now,
+        required: true
+    },
+    content: {
+        type: String,
+        required: true
+    },
+    photo: {
+        type: String,
```

Comment:
Tu chyba nie będzie type: String, tutaj będziemy przechowywać zdjęcie - plik jpg, png itp.
Ale do tego potrzebny jest skonfigurowany multer - jeśli się nie mylę.
Narazie chyba można to zostawić puste.

Reply:
multer zdjęcia koduje do base64 i przechowuje je jako String.

Reply:
ok, dobrze wiedzieć :)